mobile-app/app/screens/AppNavigator/screens/Loans/hooks/VaultShare.ts

```diff
@@ -1,5 +1,6 @@
 import BigNumber from 'bignumber.js'
 
 export function useVaultShare (collateralAmount: BigNumber, factor: BigNumber, price: BigNumber, totalCollateralValue: BigNumber): BigNumber {
-  return new BigNumber(collateralAmount ?? 0).multipliedBy(price ?? 0).multipliedBy(factor ?? 0).dividedBy(totalCollateralValue ?? 1).times(100)
+  const vaultShare = new BigNumber(collateralAmount ?? 0).multipliedBy(price ?? 0).multipliedBy(factor ?? 0).dividedBy(totalCollateralValue ?? 1)
```

Comment:
i think none of the params are optional, does `??` checks for anything?

Reply:
Oh yeah I can remove it. It's just for an earlier version I hadn't updated

Reply:
updated